pkgs/dart_mcp/lib/src/client/client.dart

```diff
@@ -264,6 +264,11 @@ base class ServerConnection extends MCPBase {
   ///
   /// The client must call [notifyInitialized] after receiving and accepting
   /// this response.
+  ///
+  /// Throws a [StateError] if initialization fails for unknown reasons (usually
+  /// the server connection closes prematurely due to misconfiguration). To
+  /// debug these errors you should pass a `protocolLogSink` when creating these
+  /// connection
```

Comment:
r/connection/connections
or r/these/this. Whichever matches the intent of the comment.